adal/src/telemetry/java/com/microsoft/aad/adal/BrokerEvent.java

```diff
@@ -33,6 +34,9 @@
  */
 
 final class BrokerEvent extends DefaultEvent {
+
+    private static boolean sEnableStackTraces = true;
```

Comment:
>true [](start = 48, length = 4)

were we going to add this to a config? also, should this be false by default?

Reply:
For the dogfood build, I'll plan to leave this `true` so that Teams doesn't need to tweak our internal settings.

If/when we produce a prod-signed build this field should be disabled.